### PR TITLE
[12.0][MIG] account_payment_widget_amount

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
This PR migrates the following two modules from #249 :

*account_move_line_auto_reconcile_hook
*account_payment_widget_amount

**account_move_line_auto_reconcile_hook**
Adds a hook to the function auto_reconcile_lines account move line model.

**account_payment_widget_amount**
Adds a popover widget in the invoice form view when there are outstanding payments that allows the user to select the amount of the selected payment that wants to assign to the invoice.

### CODE REVIEW

File: account_payment_widget_amount/static/src/xml/account_payment.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+                               data-type="number" value="" placeholder="0.00"/>
+                    </td>
+                    <td t-if="position == 'after'"><t t-esc="currency"/></td>
+                </tr>
+            </table>
+        </div>
+        <button class="btn btn-xs btn-primary js_payment_amount pull-right" t-att-id="id" style="margin-top:5px; margin-bottom:5px;">Apply</button>
+    </t>
+
+</templates>
```
Comment (etobella) [inline account_payment_widget_amount/static/src/xml/account_payment.xml]: Missing intro

**Review by etobella — APPROVED** [review]
Great Addition
:+1: 
Functional test and it works as expected :smile: 
Just a small comment

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Technical and functional review.

**Comment by andres-pcg** [comment]
Hello guys, 

Will this module be migrated to v13?

**Pull request merged**